src/main/java/org/apache/datasketches/tuple/arrayofdoubles/DirectArrayOfDoublesQuickSelectSketch.java

```diff
@@ -94,19 +115,52 @@ class DirectArrayOfDoublesQuickSelectSketch extends ArrayOfDoublesQuickSelectSke
     setRebuildThreshold();
   }
 
+  private static final boolean validate1(
```

Comment:
i feel like we can probably come up with better names :)  but not a blocker, either.

Reply:
fixed.